filament/include/filament/IndirectLight.h

```diff
@@ -342,6 +348,80 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
     /** @deprecated use static versions instead */
     UTILS_DEPRECATED
     math::float4 getColorEstimate(math::float3 direction) const noexcept;
+
+    /**
+     * Gets IBL type.
+     */
+    IndirectLight::Mode getTechnique() const noexcept;
```

Comment:
either `Mode` should be called `Technique`, or this should `getMode`. I think I prefer `getMode()`. Also the comment should be more descriptive and not use the word "type". (technique vs mode vs type, it's confusing).

Actually, I wonder if we should just call this `ProxyType` and `getProxyType()`.